nectar/config.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
+from builtins import object
 
 import os
 import tempfile
 
 import requests.auth
 
+temp_files = []
```

Comment:
I'm afraid there is a use case when we run this in multiple threads with multiple DownloaderConfig instances and I think this could lead to issue from one removing temp_files of another and etc.

I will try to investigate what causes this error and report back if I'll find anything.